examples/webgl_loader_sea3d.html

```diff
@@ -47,6 +49,9 @@
 		<script src="js/loaders/sea3d/SEA3DLZMA.js"></script>
 		<script src="js/loaders/sea3d/SEA3DLoader.js"></script>
 
+		<script src="js/libs/draco/draco_decoder.js"></script>
+		<script src="js/loaders/sea3d/SEA3DDraco.js"></script>
```

Comment:
Both Draco and LZMA on the same asset?

Reply:
LZMA is used for content `( Animations, Mesh ( Not geometry ), Sprite, Lines, Skeleton, Custom Data ... )` Draco only for geometry.